Sources/BSplineLib/VectorSpaces/weighted_vector_space.inc

```diff
@@ -80,26 +80,14 @@ WeightedVectorSpace<dimensionality>::
   }
 #endif
   Coordinate maximum_distance{};
-  Weight minimum_weight{std::numeric_limits<Weight::Type_>::max()};
+  Weight minimum_weight{std::numeric_limits<Weight>::max()};
   for (HomogeneousCoordinate_ const& homogeneous_coordinate :
        Base_::coordinates_) {
-    maximum_distance =
-        std::max(utilities::std_container_operations::TwoNorm(
-                     Project(homogeneous_coordinate)),
-                 maximum_distance,
-                 std::bind(utilities::IsLess<typename Coordinate::Name_,
-                                             typename Coordinate::Type_>,
-                           _1,
-                           _2,
-                           tolerance));
-    minimum_weight = std::min(
-        Weight{homogeneous_coordinate[dimensionality].Get()},
-        minimum_weight,
-        std::bind(
-            utilities::IsLess<typename Weight::Name_, typename Weight::Type_>,
-            _1,
-            _2,
-            tolerance));
+    maximum_distance = std::max(utilities::std_container_operations::TwoNorm(
```

Comment:
TBH I don't understand what you are doing here...

Reply:
Originally I wanted to just remove this, but it seems like it is used in remove_knots / reduce_degree to measure some relative tolerance